walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -38,23 +39,23 @@
     }
 
     Collection<String> attendees = request.getAttendees();
```

Comment:
I agree with you here. 
However, `getAttendees()` is a function that comes with `MeetingRequest`, which came with this problem set. To gain some insight into what this would look like in a real production code-base, would it be acceptable to rename a function like this? I guess it would require a separate CL and approval?

Reply:
Interesting that they already implemented `getAttendees()`. IMO this may have been a slight mistake on their part. I wouldn't try to change it at this point because I'm sure that a lot of other people now depend on that function being named `getAttendees()`.

I like your suggestion to rename attendees to requiredAttendees, I think that clears things up a bit.